zerver/webhooks/github/view.py

```diff
@@ -557,17 +557,22 @@ def api_github_webhook(
         branches: Optional[str]=REQ(default=None),
         user_specified_topic: Optional[str]=REQ("topic", default=None)) -> HttpResponse:
     event = get_event(request, payload, branches)
-    if event is not None:
-        subject = get_subject_based_on_type(payload, event)
-        body_function = get_body_function_based_on_type(event)
-        if 'include_title' in signature(body_function).parameters:
-            body = body_function(
-                payload,
-                include_title=user_specified_topic is not None,
-            )
-        else:
-            body = body_function(payload)
-        check_send_webhook_message(request, user_profile, subject, body)
+    if event is None:
+        return json_success()
```

Comment:
Probably worth adding a comment that `get_event()` returns none for events we don't want to send a notification for?

Reply:
done